addons/sale_timesheet/models/project.py

```diff
@@ -249,25 +298,52 @@ def get_panel_data(self):
         panel_data = super(Project, self).get_panel_data()
         return {
             **panel_data,
+            'analytic_account_id': self.analytic_account_id.id,
+            'sold_items': self._get_sold_items(),
             'profitability_items': self._get_profitability_items()
         }
 
+    def _get_sale_order_lines(self):
+        sale_orders = self.sale_order_id | self.tasks.sale_order_id
+        return self.env['sale.order.line'].search([('order_id', 'in', sale_orders.ids), ('is_service', '=', True)], order='id asc')
+
+    def _get_sold_items(self):
+        sols = self._get_sale_order_lines()
+        number_sale_orders = len(self.sale_order_id | self.tasks.sale_order_id)
+        sold_items = {
+            'allow_billable': self.allow_billable,
+            'data': [],
+            'number_sols': len(sols),
+            'total_sold': 0,
+            'effective_sold': 0,
+            'company_unit_name': self.env.company.timesheet_encode_uom_id.name
+        }
+        for sol in sols:
+            name = sol.order_id.name + " - " if number_sale_orders > 1 else ""
+            name += sol.name[:30] + '...' if len(sol.name) > 30 else sol.name
```

Comment:
I think if the `number_sale_orders > 1` then we could directly use the name_get of the SOL.
What do you think ?